cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A protocol for implementing high performance clifford tableau evolutions
+ for Clifford Simulator."""
+
+from typing import Iterable
+
+from cirq.sim.clifford.stabilizer_state_ch_form import StabilizerStateChForm
+
+
+class ActOnStabilizerCHFormArgs:
+    """State and context for having an operation act on a stabilizer state in
```

Comment:
nit: shorten this to make it single line.
E.g. it could be simply: "A stabilizer state in CH form for the act_on protocol."

Reply:
Done